utils/utils.go

```diff
@@ -161,3 +213,18 @@ func ContainsString(s []string, e string) bool {
 	}
 	return false
 }
+
+// Convert any bech32 to stride address
+func ConvertAddressToStrideAddress(address string) string {
```

Comment:
airdrop record key is a combination of airdrop identifier and stride address.
When we set user allocations, we basically convert all user addresses into stride addresses so that for example, multiple identical addresses for different chains can't be included in the same airdrop.